search/api.py

```diff
@@ -162,7 +167,15 @@ def search_percolate_queries(program_enrollment_id):
         django.db.models.query.QuerySet: A QuerySet of PercolateQuery matching the percolate results
     """
     conn = get_conn()
-    result = conn.percolate(get_default_alias(), USER_DOC_TYPE, id=program_enrollment_id)
+    enrollment = ProgramEnrollment.objects.get(id=program_enrollment_id)
+    doc = serialize_program_enrolled_user(enrollment)
```

Comment:
This concerns me slightly because we have identified that we spend a lot of time indexing in the serialization of the enrollment and adding another call to `serialize_program_enrolled_user` here increases the overhead of the tasks `index_program_enrolled_users` and `index_users` in `search/tasks.py`.  I worry that this change will effectively double the runtime of those tasks and also increase load on the DB. because we're hitting it twice for the same data for the same enrollment.